fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/player/DropItemCallback.java

```diff
@@ -0,0 +1,34 @@
+package net.fabricmc.fabric.api.event.player;
```

Comment:
The license should be added at the head of the files

Reply:
Licenses don't seem to be on any of the existing callback classes-- not sure if that's intentional or not.